docs/management_api_v2.yml

```diff
@@ -44,6 +44,182 @@ responses:
       $ref: "#/definitions/Error"
 
 paths:
+  /deployments/releases:
+    get:
+      operationId: List Releases with pagination
+      tags:
+        - Management API
+      security:
+        - ManagementJWT: []
+      summary: |
+        List releases
+      description: |
+        Returns a collection of releases, allows filtering by release name and sorting
+        by name or last modification date.
+      parameters:
+        - name: name
+          in: query
+          description: Release name filter.
```

Comment:
Are we planning to support search for name as it was in the old endpoint, or just name prefix as in the new UX design?

Reply:
implemented name prefix